docs/metrics.md

```diff
@@ -52,15 +52,16 @@ rules:
 
 ### EventSource
 
-- `event_service_running_total` - How many configured events in the EventSource
```

Comment:
in Argo Workflow the Prometheus HELP contains deep-links to the docs, you could do this same here, allowing you to change the help with a release